src/domain/home/hero/HomeHero.tsx

```diff
@@ -25,7 +27,13 @@ const HomeHero = ({
       <div className={styles.heroContainer}>
         <div className={styles.hero}>
           <h1>{t('appName')}</h1>
-          <p> {t('homePage.hero.descriptionText')}</p>
+          <p className={styles.bodyXl}>{t('homePage.hero.descriptionText')}</p>
+          {Config.featureFlagShowCoronaInfo && (
```

Comment:
What if you used a generic name for this, for example `showFrontPageAlert` or something like that? Then it can be reused for other things in the future. 

I'm watching this project because I can learn from the improvements you're making - one of the advantages of open source!

Reply:
To make the notification at least somewhat reusable would be relatively difficult when contrasted with the scope of this task. Naming the feature flag in a generic way could be confusing. First, it would be difficult to say what notification it would toggle. Second, it would imply functionality that's not implemented. Using a name that's not reflective of the current state, but instead a future vision, can be really confusing.

Moreover, the feature flag is a stretch in scope in itself and only meant as an escape hatch in case there's not enough development bandwith to remove the notification when events are resumed.

Thank you for the suggestion!

Reply:
Everything you say makes sense! I do hope you'll be able to toggle it of and remove it soon..